From de0c1f833c12a992af5c7daebe1705cd2c72f743 Mon Sep 17 00:00:00 2001
From: Stephen Kitt <steve@sk2.org>
Date: Mon, 1 Nov 2021 17:25:07 +0100
Subject: [PATCH] Fix unbalanced parentheses in FAudio.c

Commit b5916945b90d ("Dont assert on WMA buffer and
FAUDIO_FORMAT_EXTENSIBLE type") introduced unbalanced parentheses,
which breaks builds using HAVE_WMADEC.

Given the change made in the aforementioned commit, this balances
parentheses by adding a third closing parenthesis after the reworked
conditions.

Signed-off-by: Stephen Kitt <steve@sk2.org>
---
 src/FAudio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/FAudio.c b/src/FAudio.c
index d3b0b4d8..ff373ef8 100644
--- a/src/FAudio.c
+++ b/src/FAudio.c
@@ -2455,7 +2455,7 @@ uint32_t FAudioSourceVoice_SubmitSourceBuffer(
 #ifdef HAVE_WMADEC
 	FAudio_assert(	(voice->src.wmadec != NULL && (pBufferWMA != NULL ||
 					(voice->src.format->wFormatTag == FAUDIO_FORMAT_XMAUDIO2 ||
-					 voice->src.format->wFormatTag == FAUDIO_FORMAT_EXTENSIBLE)) ||
+					 voice->src.format->wFormatTag == FAUDIO_FORMAT_EXTENSIBLE))) ||
 			(voice->src.wmadec == NULL && (pBufferWMA == NULL && voice->src.format->wFormatTag != FAUDIO_FORMAT_XMAUDIO2))	);
 #endif /* HAVE_WMADEC */
 
